app/app.py

```diff
@@ -172,9 +173,8 @@ async def authenticate(payload: RequestModel):
         )
     except ValidationError:
         logging.exception(f"Validation error on ResponseModel for user={username}.")
-        # TODO: This will NOT be handled by the exception handler. It will return a "detail" field which we do not want. Same in /readme (and other endpoints)
-        raise HTTPException(
-            status_code=500, detail="Internal Server Error. Please try again later."
+        raise Exception(
+            "Internal Server Error. Please try again later."
```

Comment:
Raise `PESUAcademyError` here instead, since an error here will only occur because of something wrong returned from `pesu.py`